app/controllers/solidus_klarna_payments/api/sessions_controller.rb

```diff
@@ -2,9 +2,11 @@
 
 module SolidusKlarnaPayments
   module Api
-    class SessionsController < ::Spree::Api::BaseController
+    class SessionsController < BaseController
```

Comment:
I have [removed SolidusKlarnaPayments::Api::BaseController](https://github.com/solidusio-contrib/solidus_klarna_payments/pull/51/commits/9863ce3fa49986bd00bc35cfeef05b5ada010e2e) because it inherited from StoreController and broke the store without the frontend

Reply:
hmm, however inheriting from ::Spree::Api::BaseController breaks it too as we are not providing an API key and helper methods like ``try_spree_current_user` are not available 🤔

Reply:
This API uses the order, so we have to change the JS in order to provide the order token so the API will load the right order.

https://github.com/solidusio-contrib/solidus_klarna_payments/pull/50/files/0c2de5cb25f5447dceff3f0aec49bab2bb70ff1e#diff-380f724921f65f3c3e9cfd6e01a3b40e249f60ff0b5ac7fc2ab08cf02dbe11f6R43-R45 

This is not the right way to load the order because allows to any user to generate a klarna session for any orders, without checking the owner of the order, right?